### PR TITLE
PROGRAMMES-6753 - added default titles to clip blocks

### PR DESCRIPTION
**Pull request by Bobkidbob**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6753

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by Bobkidbob** [comment]
Should probably try to get rid of the repetition tomorrow. The same code is also used in the galleries block.

File: src/Ds2013/Presenters/Domain/ContentBlock/Clip/ClipStandalone/clip_standalone.html.twig [hunk 1]
```diff
@@ -6,6 +6,18 @@
     {% endif %}
 {% endblock %}
 
+{% block title %}
```
Comment (cisco87) [inline src/Ds2013/Presenters/Domain/ContentBlock/Clip/ClipStandalone/clip_standalone.html.twig]: I think you could put them in a shared/partial and pass as argument 1 or 2 like {% include 'shared_name.html.twig' with {'number_of_clips': 1 or 2} %} and read it on the tr('clips', xyz), at least I guess we don't have a duplicate template? what do you think? they look quite identical to me

**Comment by Bobkidbob** [comment]
Pretty sure that code review was before the refactor, so waiting for another.

**Pull request merged**